sql/parser/sql.y

```diff
@@ -826,6 +828,14 @@ grant_stmt:
     $$ = &Grant{Privileges: $2, Targets: $4, Grantees: NameList($6)}
   }
 
+// REVOKE privileges ON privilege_target FROM grantee_list
```

Comment:
I've seen this in a few places in our grammar. Granted, it's not particularly critical here.